### PR TITLE
Dev/issue 11097 clipboard export improve

### PR DESCRIPTION
**Pull request by mcantelon**
* Allow unauthenticated export jobs to be deleted only after job is complete
* Fixed Internet Explorer-specific clipboard issue

### CODE REVIEW

File: apps/qubit/modules/default/actions/alertsComponent.class.php [hunk 1]
```diff
@@ -65,8 +65,11 @@ public function execute($request)
           );
           $alertType = $alertTypes[$job->getStatusId()];
 
-          // Add as conditional alert
+          // If job is complete, allow it to be deleted by the user
           $deleteUrl = $this->context->controller->genUrl('jobs/delete?id='. $job->id);
+          $deleteUrl = ($job->getStatusId() == QubitTerm::JOB_STATUS_COMPLETED_ID) ? $deleteUrl : null;
+
+          // Add as conditional alert
           array_push($this->conditionalAlerts, array('type' => $alertType, 'message' => $message, 'deleteUrl' => $deleteUrl));
```
Comment (sbreker) [inline apps/qubit/modules/default/actions/alertsComponent.class.php:73]: Is it OK that we could push an alert where $deleteUrl is null?

Comment (mcantelon) [reply apps/qubit/modules/default/actions/alertsComponent.class.php:73]: This is for controlling what happens when the X (to close a notification) is clicked. If the $deleteUrl is null then clicking X will have the standard behaviour of dismissing the notification.

Comment (sbreker) [reply apps/qubit/modules/default/actions/alertsComponent.class.php:73]: Sounds good!

Comment (sbreker) [reply apps/qubit/modules/default/actions/alertsComponent.class.php:73]: Looks good!

**Pull request merged**